libnetwork/drivers/bridge/port_mapping_linux.go

```diff
@@ -675,14 +674,7 @@ func bindSCTP(cfg portBindingReq, port int) (_ portBinding, retErr error) {
 		syscall.SetsockoptInt(sd, syscall.IPPROTO_IPV6, syscall.IPV6_V6ONLY, 1)
 	}
 
-	options := sctp.InitMsg{NumOstreams: sctp.SCTP_MAX_STREAM}
-	if _, _, errno := syscall.Syscall6(syscall.SYS_SETSOCKOPT,
```

Comment:
Regression point was https://github.com/moby/moby/commit/b3fabedecc0f408f674972d87803f41faedf4de8 correct? So from that, there's low risk of that getting into any of the release branches

- https://github.com/moby/moby/pull/48132

Reply:
Yup, only v28.0.0 is affected.